clients/admin-ui/cypress/fixtures/plus_health.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This payload was updated a little in the recent fidepslus 2.0 merge. This is what it looks like now.

```json
{
    "core_fides_version": "2.2.0",
    "fidesplus_server": "healthy",
    "system_scanner": {
        "enabled": false,
        "cluster_health": null,
        "cluster_error": null,
    },
}
```

Does it make sense to update this now?